src/containers/CreateNote/CreateNote.js

```diff
@@ -8,44 +8,58 @@ class CreateNote extends React.Component {
     super(props);
     this.state = {
       title: "",
-      noteItems: [
-        { id: uuid(), text: "default" },
-        { id: uuid(), text: "default" }
-      ]
+      noteItems: [],
+      currentFocus: null
```

Comment:
When I added keys to all elements that are being mapped, I lost focus on the re-render. Using a currentFocus property , we can know what the user last focused on  and after the element has mounted, it will autofocus to it.